atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -309,13 +314,15 @@ private void saveSweepResults(final SweepProgressRowResult progress,
     private void saveIntermediateSweepResults(final SweepProgressRowResult progress,
                                               final byte[] nextStartRow,
                                               final long cellsDeleted,
-                                              final long cellsExamined) {
+                                              final long cellsExamined,
+                                              final long minimumSweptTimestamp) {
         txManager.runTaskWithRetry(new TxTask() {
             @Override
             public Void execute(Transaction t) {
                 SweepProgressTable progressTable = tableFactory.getSweepProgressTable(t);
                 SweepProgressRow row = SweepProgressRow.of(0);
                 progressTable.putFullTableName(row, progress.getFullTableName());
+                progressTable.putMinimumSweptTimestamp(row, minimumSweptTimestamp);
```

Comment:
You should only have to write this on the first set (see the !progress.hasStartRow() check below).